src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -62,25 +62,28 @@ internal sealed class AnalysisConfigMonitor : IAnalysisConfigMonitor, IDisposabl
 
         private void OnUserSettingsChanged(object sender, EventArgs e)
         {
-            // NB assumes exception handling is done by the AnalysisRequester
-            logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
-            
-            // TODO: once connected mode is supported for all languages this event could
-            // be ignored in connected mode. See #771.
-            if (activeSolutionBoundTracker.CurrentConfiguration.Mode != SonarLintMode.Standalone)
+            if (activeSolutionBoundTracker.CurrentConfiguration.Mode == SonarLintMode.Standalone)
+            {
+                logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
```

Comment:
Previously, this log message was written always, and it says "open documents will be re-analyzed". Since that's no longer true, I've changed it and now we're either writing the Standalone message or the Connected message.